webpack/assets/javascripts/react_app/redux/consts.js

```diff
@@ -1,6 +1,8 @@
+export const STATISTICS_DATA = 'STATISTICS_DATA';
```

Comment:
Shouldn't this PR suppose to remove all those extra consts with the suffixes? (e.g. `STATISTICS_DATA_REQUEST`, `STATISTICS_DATA_SUCCESS`, `STATISTICS_DATA_FAILURE`)

Reply:
These consts are still needed for the reducer